lib/guesses.rb

```diff
@@ -6,7 +6,7 @@ def letter_is_present?(guess, word)
 
   def check_input_is_valid?(guess)
     alphabet = ("a".."z").to_a.join
-    alphabet.include?(guess)
+    alphabet.include?(guess) || guess == "nutella"
```

Comment:
Good job for removing `guess == "nutella"` from `Display`. However this method is now lying. It's not just checking if the input is valid. It's also checking for something else. How can you change that?